docs/setup-and-installing.md

```diff
@@ -35,25 +35,29 @@ This guide explains how to set up the SpatialOS Unreal GDK and the Unreal Engine
 To build the SpatialOS Unreal GDK module you need the following installed:
 * Windows 10, with Command Prompt or PowerShell as your terminal
 * [Git for Windows](https://gitforwindows.org)
-* [SpatialOS version 13](https://docs.improbable.io/reference/13.1/shared/get-started/setup/win)
+* [SpatialOS version 13.1](https://console.improbable.io/installer/download/stable/latest/win).
+This installs:<br>
+		- the [`spatial` CLI](https://docs.improbable.io/reference/13.0/shared/spatial-cli-introduction)<br>
```

Comment:
I _think_ these links should go to 13.1, as that's what we're now pinned to. Not sure who would be able to confirm for definite.

Reply:
good spot